hw/ip/otbn/doc/_index.md

```diff
@@ -439,6 +439,25 @@ outer_body:
 
 ## Design Details
 
+### Memories
+
+The OTBN processor core has access to two dedicated memories:
+an instruction memory (IMEM), and a data memory (DMEM).
+Each memory is 4 kiB in size.
+
+The memory layout follows the Harvard architecture.
+Both memories are byte-addressed, with addresses starting at 0.
+
+The instruction memory (IMEM) is 32b wide and provides the instruction stream to the OTBN processor;
+it cannot be read or written from user code through load or store instructions.
```

Comment:
I think this should probably be "read or written by user code". Or "read from or written to by user code", but that's a bit of a mouthful.